features/insights-results-aggregator-mock/basic_rest_api.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Basic REST API endpoints provided by Insights Results Aggregator Mock
+
+
+  Scenario: Check if the main endpoint is reachable
+    Given the system is in default state
+      And REST API service hostname is localhost
+      And REST API service port is 8080
+      And REST API service prefix is /api/insights-results-aggregator/v1
```

Comment:
Is there some reason to separate this 3 "given" sentences instead of having only one like "REST API base endpoint is ...."?

Reply:
yes, at least it split "given" and "when" steps clearly